app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupActivity.java

```diff
@@ -7,12 +7,11 @@
 import android.widget.Button;
 import android.widget.ProgressBar;
 
-import com.google.gson.Gson;
```

Comment:
All removed imports were removed automatically from the refactoring tools when I moved the models to DataModels folder.